flopy/utils/postprocessing.py

```diff
@@ -637,9 +637,9 @@ def get_specific_discharge(
     if classical_budget:
         # get saturated thickness (head - bottom elev for unconfined layer)
         if head is None:
-            sat_thk = modelgrid.remove_confining_beds(modelgrid.thick)
```

Comment:
Even though it doesn't matter the local variable `sat_thk` could also be renamed `saturated_thickness` for clarity.

Reply:
@jdhughes-usgs refactored `sat_thk` to `saturated_thickness`